src/vscode/superbol-vscode-platform/superbol_tasks.ml

```diff
@@ -111,9 +113,9 @@ let cobc_execution ?config attributes =
       ~mk:((^) "-fformat=") |>
     attr_bool_flag "for-debug" ~attributes
       ~ok:(fun args -> "-fsource-location" :: "-ftraceall" ::
-                       "-g" ::
-                       "-Q" :: "--coverage" ::
-                       "-A" :: "--coverage" :: args) |>
+                       "-g" :: args) |>
+    attr_bool_flag "enable-coverage" ~attributes
+      ~ok:(fun args -> "--coverage" :: args) |>
```

Comment:
Don't we need `-A --coverage` and `-Q --coverage` still (not sure…)?
```
$ cobc --help
…
  -A <options>          add <options> to the C compile phase
  -Q <options>          add <options> to the C link phase
  --coverage            instrument generated binaries for coverage
…
```

Reply:
No they are equivalent, maybe these `-Q --coverage -A --coverage` comes from times where `--coverage` was not a direct GnuCOBOL options, and it had to be passed to the C compiler and linker by hand